### ISSUE TITLE
Date of visit or admission gets set to current date when changing place of stay 

### ISSUE THREAD

**Issue by kwa20** [hzi-braunschweig/SORMAS-Project#4480]
### Bug Description
When the place of a case gets changed from hospital to home, the current hospitalization is being migrated to the previous hospitalizations. However, the date of visit or admission is also set to the current date.

This entry can also not be deleted since the field becomes disabled when the current place of stay is not hospital.

Furthermore, the additional entry produces an incorrect additional hospitalization entry when transferring data to SurvNet.
### Steps to Reproduce
1. Create a case whose current place of stay is hospital.
2. Enter various hospitalization fields. Save.
3. Set the place of stay to anything but hospital. Transfer case.
4. Check hospitalization information.

### Expected Behavior
When transferring the case, the current hospitalization should merely be migrated without setting the date of visit or admission to the current date.
### Screenshots
<img width="483" alt="hospitalizationBug" src="https://user-images.githubusercontent.com/70317594/108534954-03ace980-72db-11eb-9101-61f2fc86836f.PNG">
<img width="437" alt="hospitalizationBug2" src="https://user-images.githubusercontent.com/70317594/108534960-0576ad00-72db-11eb-8369-999e65a3509b.PNG">

### System Details
* Device:
* SORMAS version: 1.56.1
* Android version/Browser: firefox
* Server URL: test1402.sormas-oegd.de
* User Role: admin, national user

**Comment by MateStrysewske** [comment]
@kwa20 If I understood you correctly, the date should basically be deleted, right? That would make the most sense because "Home" is not a hospitalization in the first place.

**Comment by kwa20** [comment]
@MateStrysewske yes, it also does not seem to be a residue of a previous hospitalization because a previous date period might not match the date that is left behind in the date of admission.

**Comment by AndyBakcsy-she** [comment]
Verified ticket on the latest version of sormas 1.58.0-snapshot on test.sormas.netzlink.com
